python/mxnet/metric.py

```diff
@@ -437,13 +437,29 @@ def feval(label, pred):
 # pylint: enable=invalid-name
 
 def create(metric, **kwargs):
-    """Create an evaluation metric.
+    """Creates an evaluation metric from metric name(s) or instance(s) of EvalMetric
```

Comment:
Can't we just say "Creates evaluation metrics from metric names or instances of EvalMetric"